services/app-api/serverless.yml

```diff
@@ -82,19 +64,10 @@ provider:
           Resource:
             - ${self:custom.tableArn}
             - ${self:custom.atomicCounterTableArn}
-        - Effect: "Allow" #Warmup permissions
-          Action:
-            - lambda:InvokeFunction
-          Resource:
-            - !Sub arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:${self:service}-${opt:stage, self:provider.stage}-*
   environment:
     tableName: ${self:custom.tableName}
     atomicCounterTableName: ${self:custom.atomicCounterTableName}
 
-associateWaf:
```

Comment:
This needs to be under 'custom', not top level, per the docs.  I don't think this has been working...